inc/Engine/Common/JobManager/APIHandler/APIClient.php

```diff
@@ -47,14 +47,28 @@ public function add_to_queue( string $url, array $options ): array {
 		 */
 		$url = apply_filters( 'rocket_saas_api_queued_url', $url );
 
+		$blocking = true;
+
+		if ( isset( $options['blocking'] ) ) {
+			$blocking = $options['blocking'];
+
+			unset( $options['blocking'] );
+		}
```

Comment:
When the `WP_ROCKET_DEBUG` is not set at all, so it'll be null and we will not get into this block at all then $blocking will equal "true" but we need it to be "false"

I think to solve this quickly we can change the default value for the constant to be false not null like:

```
rocket_get_constant( 'WP_ROCKET_DEBUG', false )
```

Reply:
good catch, I updated the passed value in the APIClient